paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/PaymentSheetTopBarTest.kt

```diff
@@ -0,0 +1,112 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.platform.SoftwareKeyboardController
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.unit.dp
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.paymentsheet.R
+import com.stripe.android.utils.HackyComposeTestRule
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.never
+import org.mockito.kotlin.verify
+
+@OptIn(ExperimentalComposeUiApi::class)
```

Comment:
Guessing this is in relation to the keyboard stuff. Should we just opt in at the module level? I find these littered annotations annoying. But that's my bike shed. Feel free to ignore it.